modules/virt-inferfromvolume-labels.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-specifying-instance-preference.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="inferfromvolume-labels_{context}"]
+= Setting the inferFromVolume labels
+
+Use the following labels on your PVC, data source, or data volume to instruct the inference mechanism which instance type, preference, or both to use when trying to boot from a volume.
+
+* A cluster-wide instance type: `instancetype.kubevirt.io/default-instancetype` label.
+* A namespaced instance type: `instancetype.kubevirt.io/default-instancetype-kind` label. Defaults to the `VirtualMachineClusterInstancetype` label if left empty.
+* A cluster-wide preference: `instancetype.kubevirt.io/default-preference` label.
+* A namespaced preference: `instancetype.kubevirt.io/default-preference-kind` label. Defaults to `VirtualMachineClusterPreference` label, if left empty.
+
+.Prerequisites
+
+* You must have an instance type, preference, or both on the cluster.
+
+.Procedure
+
+* To apply a label to a data source, use `oc label`. The following command applies a label that points to a cluster-wide instance type:
+
++
+[source,terminal]
+----
+$ oc label DataSource foo instancetype.kubevirt.io/default-instancetype=<my-instancetype>
```

Comment:
Change `-` to `_` in the [user-replaceable values](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values) throughout
```suggestion
$ oc label DataSource foo instancetype.kubevirt.io/default-instancetype=<my_instancetype>
```